shallow_backup/prompts.py

```diff
@@ -15,13 +16,21 @@ def path_update_prompt(config):
 	current_path = config["backup_path"]
 	print_path_blue("Current shallow-backup path:", current_path)
 	if prompt_yes_no("Would you like to update this?", Fore.GREEN):
-		print_green_bold("Enter relative or absolute path:")
-		abs_path = expand_to_abs_path(input())
-		print_path_blue("\nUpdating shallow-backup path to:", abs_path)
-		mkdir_warn_overwrite(abs_path)
-		move_git_repo(current_path, abs_path)
-		config["backup_path"] = abs_path
-		write_config(config)
+		while True:
+			print_green_bold("Enter relative or absolute path:")
+			abs_path = expand_to_abs_path(input())
+
+			if os.path.isfile(abs_path):
+				print_path_red('New path is an existing file:', abs_path)
+				print_red_bold('Please enter a directory.\n')
+				continue
+
+			print_path_blue("\nUpdating shallow-backup path to:", abs_path)
+			mkdir_warn_overwrite(abs_path)
+			move_git_repo(current_path, abs_path)
+			config["backup_path"] = abs_path
+			write_config(config)
+			break
```

Comment:
```suggestion
			return
```

Reply:
Functionally, it does the same thing, but this makes the control flow more clear, imo.

Reply:
cb2b47ea9e5e8736783e610ae459e30a8a6bb305